### PR TITLE
 Feature - Kotlin Code Style

### PR DESCRIPTION
**Pull request by chahine**

### CODE REVIEW

File: configs/codestyles/ProlificAndroid.xml [hunk 1]
```diff
@@ -107,13 +107,10 @@
     <option name="PACKAGES_TO_USE_STAR_IMPORTS">
       <value />
     </option>
-    <option name="NAME_COUNT_TO_USE_STAR_IMPORT" value="999" />
-    <option name="NAME_COUNT_TO_USE_STAR_IMPORT_FOR_MEMBERS" value="999" />
+    <option name="NAME_COUNT_TO_USE_STAR_IMPORT" value="2147483647" />
```
Comment (quentin41500) [inline configs/codestyles/ProlificAndroid.xml:110]: Is this going to be enough Oo?

**Review by quentin41500 — APPROVED** [review]
LGTM

File: configs/codestyles/ProlificAndroid.xml [hunk 2]
```diff
@@ -107,13 +107,10 @@
...
+    <option name="NAME_COUNT_TO_USE_STAR_IMPORT_FOR_MEMBERS" value="2147483647" />
     <option name="IMPORT_NESTED_CLASSES" value="true" />
   </JetCodeStyleSettings>
-  <MarkdownNavigatorCodeStyleSettings>
-    <option name="RIGHT_MARGIN" value="72" />
-  </MarkdownNavigatorCodeStyleSettings>
   <Objective-C-extensions>
```
Comment (jplipata) [inline configs/codestyles/ProlificAndroid.xml:114]: Dumb question: Why are there Objective-C extensions in Kotlin code style?

File: configs/codestyles/ProlificAndroid.xml (hunk 1 above)
Comment (chahine) [reply configs/codestyles/ProlificAndroid.xml:110]: This is MAX_INT to force the developer to never use wildcard import

File: configs/codestyles/ProlificAndroid.xml (hunk 2 above)
Comment (chahine) [reply configs/codestyles/ProlificAndroid.xml:114]: This is the default code style used by IntelliJ/Android Studio

**Pull request merged**